File: 05. Percabangan/E. If Then Case/e_if_then_case.pas
program e_if_then_case;
var N : longint;
begin
  readln(N);
  if N > 9999 then begin
      writeln('puluhribuan');
  end
  else if N > 999 then begin
    writeln('ribuan');
  end
  else if N > 99 then begin
    writeln('ratusan');
  end
  else if N > 9 then begin
    writeln('puluhan');
  end
  else begin
    writeln('satuan');
  end;
end.

File: 05. Percabangan/E. If Then Case/e_if_then_case.java
import java.util.Scanner;

class e_if_then_case {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        scan.close();
        if (N > 9999){
            System.out.println("puluhribuan");
        } else if (N > 999){
            System.out.println("ribuan");
        } else if (N > 99){
            System.out.println("ratusan");
        } else if (N > 9){
            System.out.println("puluhan");
        } else{
            System.out.println("satuan");
        }
    }
}
